app/models/post.rb

```diff
@@ -0,0 +1,5 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :likes
+  has_many :comments
+end
```

Comment:
- [ ] Good job working on this but you did not include a method that returns the five most recent comments.
- [ ] Also, please kindly include a method that increments the post_counter after a post is created saved just like you did in the Comment model.